cmd/consensus.cpp

```diff
@@ -633,17 +627,26 @@ Status difficulty_test(const nlohmann::json& j, const std::optional<ChainConfig>
     }
 }
 
-bool exclude_test(const fs::path& p, const fs::path& root_dir) {
-    return as_range::any_of(kExcludedTests,
-                            [&p, &root_dir](const std::filesystem::path& e) -> bool { return root_dir / e == p; });
+bool exclude_test(const fs::path& p, const fs::path& root_dir, bool include_slow_tests) {
+    const auto path_fits = [&p, &root_dir](const fs::path& e) { return root_dir / e == p; };
+    return as_range::any_of(kFailingTests, path_fits) ||
+           (!include_slow_tests && as_range::any_of(kSlowTests, path_fits));
 }
 
 int main(int argc, char* argv[]) {
+    const auto begin{std::chrono::steady_clock::now()};
+
     CLI::App app{"Run Ethereum consensus tests"};
+
     std::string evm_path{};
     app.add_option("--evm", evm_path, "Path to EVMC-compliant VM");
     std::string tests_path{SILKWORM_CONSENSUS_TEST_DIR};
-    app.add_option("--tests", tests_path, "Path to consensus tests", true)->check(CLI::ExistingDirectory);
+    app.add_option("--tests", tests_path, "Path to consensus tests", /*defaulted=*/true)->check(CLI::ExistingDirectory);
+    unsigned num_threads{std::thread::hardware_concurrency()};
```

Comment:
We should check what it returns in CI/docker. CMake had/has issue with similar context where it returns the number of hardware threads of a CPU even though it runs in a virtualized / containerized environment with number of CPU cores capped.